app/models/account.rb

```diff
@@ -36,4 +38,15 @@ def reset!
     solr_endpoint.reset! if solr_endpoint
     fcrepo_endpoint.reset! if fcrepo_endpoint
   end
+
+  def default_cname
+    return unless name
+
+    default_host = Settings.multitenancy.default_host
+    default_host % { tenant: name.parameterize }
+  end
+
+  def canonicalize_cname
```

Comment:
A comment about what is happening and why would be helpful to me. Removing a dot at the end? Should you use \Z instead of $ ?